### DISCUSSION TITLE
AI SDK for ABAP licensing

### DISCUSSION THREAD

**Discussion by rgtripathi** [microsoft/aisdkforsapabap#18] [General]
Are there any sort of licensing  / cost implications in using AI SDK for ABAP (and ABAPGIT) for enterprises?  (understand that Azure OpenAI use would be billed as per prevalent Azure pricing in normal course).

Thx.

**Comment by gopalnair** [comment]
None. The AI SDK for ABAP is open source under MIT license. You can use it without any cost implications as far as the AI SDK usage is concerned.

Of course, as you rightly mentioned, the AI Engine services (Eg: Azure Open AI, or Open AI ) usage may incur cost depending on the usage like Finetunings, completions...etc.


Best Regards,
Gopal Nair.

**Comment by rgtripathi** [comment]
Thanks for the clarity, Gopal!

BR,
Rakesh
